input/pagecontent/reference.md

```diff
@@ -1126,31 +1128,49 @@ These conformance resources are created using FSH instance grammar. For example,
 
 <span class="caption" id="t8">Table 8. Keywords used to define Invariants</span>
 
-| Keyword | Usage | Corresponding element in ElementDefinition | Data Type | Required |
-|-------|------------|--------------|-------|----|
-| Invariant | Identifier for the invariant | constraint.key | id | yes |
-| Description | Human description of constraint | constraint.human | string  | yes |
-| Expression | FHIRPath expression of constraint | constraint.expression | FHIRPath string | no |
-| Severity | Either #error or #warning, as defined in [ConstraintSeverity](https://www.hl7.org/fhir/R4/valueset-constraint-severity.html) | constraint.severity | code | yes |
-| XPath | XPath expression of constraint | constraint.xpath | XPath string | no |
+| Keyword | Usage | Corresponding element in <br/> ElementDefinition.constraint | Data Type | Required |
+|-------------|-----------------------------------|------------|-----------------|----------------|
+| Invariant   | Identifier for the invariant      | key        | id              | yes            |
+| Description | Human description of constraint   | human      | string          | no<sup>*</sup> |
+| Expression  | FHIRPath expression of constraint | expression | FHIRPath string | no             |
+| Severity    | Either #error or #warning, as defined in [ConstraintSeverity](https://www.hl7.org/fhir/R4/valueset-constraint-severity.html) | severity | code | no<sup>*</sup> |
+| XPath       | XPath expression of constraint    | xpath      | XPath string    | no             |
 {: .grid }
 
+{%include tu-div.html%}
+Authors may also specify ElementDefinition.constraint elements via [assignment rules](#assignment-rules). This approach is particularly useful for specifying constraint extensions such as the [Best Practice](https://hl7.org/fhir/extensions/StructureDefinition-elementdefinition-bestpractice.html) extension. Invariant definitions also allow [path rules](#path-rules) and [insert rules](#insert-rules).
+
+<sup>*</sup>If the `Description` keyword is not specified, then the definition must contain an assignment rule for the `human` element. If the `Severity` keyword is not specified, then the definition must contain an assignment rule for the `severity` element.
+</div>
+
 **Example:**
 
-* Define an invariant found in US Core using FSH:
+* Define a simplified version of an invariant found in US Core using FSH Invariant keywords only:
 
   ```
-  Invariant:   us-core-8
+  Invariant:   us-core-6
   Description: "Patient.name.given or Patient.name.family or both SHALL be present"
-  Expression:  "family.exists() or given.exists()"
   Severity:    #error
+  Expression:  "family.exists() or given.exists()"
   XPath:       "f:given or f:family"
   ```
 
+{%include tu-div.html%}
+* Define a simplified version of an invariant found in US Core using FSH Invariant keywords and rules:
+
+  ```
+  Invariant:   us-core-6
+  Description: "Patient.name.given or Patient.name.family or both SHALL be present"
```

Comment:
Do you think it would be a helpful example to replace the `Description` keyword with an assignment rule on `human` in this example Invariant?

Reply:
Well... I'm a little torn on that because I want to _encourage_ people to use `Description:` since that is more consistent w/ all the other definition types. The way I put it here (using `Invariant:` and `Description:`, then rules for everything else, is what I personally consider best practice.

But if you think it would be better to show how the `Description:` can be done in a rule using `human` (since `Description:` is already demonstrated above), I can be easily convinced. As I said, I'm torn on it.

Reply:
Hmm for whatever reason, I hadn't realized/forgot that the best practice was to use `Description`, even though there is the AssignmentRule option now. I think the only way we really convey best practices is through the examples, so I think I'm good with keeping this as is.